config/v1alpha1/types_image_policy.go

```diff
@@ -143,6 +151,39 @@ type PolicyFulcioSubject struct {
 	SignedEmail string `json:"signedEmail"`
 }
 
+// PKI defines the root of trust based on Root CA(s) and corresponding intermediate certificates.
+// +openshift:enable:FeatureGate=SigstoreImageVerificationPKI
+type PKI struct {
+	// caRootsData contains base64-encoded data of a certificate bundle PEM file, which contains one or more CA roots in the PEM format.
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:MaxLength=8192
+	CertificateAuthorityRootsData []byte `json:"caRootsData"`
+	// caIntermediatesData contains base64-encoded data of a certificate bundle PEM file, which contains one or more intermediate certificates in the PEM format.
+	// caIntermediatesData requires CertificateAuthorityRoots to be set.
+	// +optional
+	// +kubebuilder:validation:MaxLength=8192
+	CertificateAuthorityIntermediatesData []byte `json:"caIntermediatesData,omitempty"`
```

Comment:
What is the benefit of separating intermediates vs just providing one complete chain?

Reply:
There was a discussion in the enhancement review where it was decided to explicitly separate intermediates and root data to avoid potential mistakes https://github.com/openshift/enhancements/pull/1658#pullrequestreview-2219422365,so we did not include the certificate-chain field.